.github/workflows/build.yaml

```diff
@@ -28,15 +28,15 @@ jobs:
                     coverage: "none"
                     extensions: "${{ env.REQUIRED_PHP_EXTENSIONS }}"
                     php-version: "${{ matrix.php-version }}"
-                    tools: composer:v2
+                    tools: composer:2.1
```

Comment:
Is this change needed?

Reply:
Yes, it seems box has some issues with composer 2.2.x

Reply:
Ok 👍